js_modules/dagit/packages/core/src/overview/OverviewSchedulesRoot.tsx

```diff
@@ -240,7 +247,7 @@ export const OverviewSchedulesRoot = () => {
         </Box>
         <Tooltip
           content="You do not have permission to start or stop these schedules"
-          canShow={!viewerHasAnyInstigationPermission}
+          canShow={anySchedulesVisible && !viewerHasAnyInstigationPermission}
```

Comment:
Fixed the case where there were zero schedules/sensors at all, and therefore zero permissioned schedules/sensors, and therefore a visible tooltip. We don't want the tooltip in this empty case.